src/Parser/DeathKnight/Frost/Modules/Features/Abilities.js

```diff
@@ -59,8 +59,8 @@ class Abilities extends CoreAbilities {
         cooldown: 180,
         castEfficiency: {
           suggestion: true,
-          recommendedEfficiency: 0.90,
-          extraSuggestion: <span>Normally you should be using this off CD.</span>,
+          recommendedEfficiency: 0.60,
+          extraSuggestion: <span>You should use this whenever you are out of Runes and Runic Power.  Alternatively, some players use this during heroism to get more casts of <SpellLink id={SPELLS.OBLITERATE.id}/>.</span>,
         },
```

Comment:
You should use the <Wrapper> instead of <span>

Reply:
I think either you forgot a word or my reading comprehension is not working today

Reply:
formatting ... 😓 
![image](https://user-images.githubusercontent.com/29204244/35115825-beb9771e-fc89-11e7-8c12-d5922c6e8d50.png)